build/app_template_files/__ReactNativeTemplateAppName__/__ReactNativeTemplateAppName__.xcodeproj/project.pbxproj

```diff
@@ -308,7 +308,6 @@
 				ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
 				INFOPLIST_FILE = __ReactNativeTemplateAppName__/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = "__CompanyIdentifier__.${PRODUCT_NAME:rfc1034identifier}";
```

Comment:
This actually didn't compile for device.  `createApp.sh` does not look at replacing __CompanyIdentifier__ in the project file, making for a bogus Bundle ID.  I normalized the Info.plist for React with the other template app setups.